tests/e2e/specs/home-spec.ts

```diff
@@ -0,0 +1,39 @@
+import HomePage from '../pages/home/home-page';
+
+const home = new HomePage();
+
+describe('Homepage test', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should render the homepage', () => {
+    cy.contains('Gallery', { matchCase: false }).should('be.exist');
+  });
+
+  it('should redirect to members page when click the explore button', () => {
+    home.getExploreButton().should('be.exist');
+    home.getExploreButton().click();
+    cy.url().should('include', '/members');
+  });
+
+  it('should redirect to collection page when click the sign in button', () => {
+    home.getSignInButton().should('be.exist');
+    home.getSignInButton().click();
+    home.getMetaMaskButton().click();
+    home.acceptMetamaskAccessRequest();
+    cy.confirmMetamaskSignatureRequest();
+    // TODO: Replace this with fixture
```

Comment:
what does this mean? is the TODO for this PR?

Reply:
I'm thinking to address this on upcoming PR when we added more test cases. 

Basically, move all the hardcoded data into a fixture config - https://docs.cypress.io/api/commands/fixture#Examples